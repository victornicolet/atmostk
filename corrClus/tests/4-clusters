1 2 3 4 5 6 7 8 9 10
9 1 -1
3 4 1
4 7 1
8 3 1
10 8 3
10 3 -4
1 2 3
2 5 2
6 5 1
9 5 -1
9 10 -2
3 7 1
9 7 -1
