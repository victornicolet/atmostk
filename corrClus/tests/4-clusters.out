9
10 8
1 2 5 6
3 4 7